test/e2e/cypress/integration/profiles/mobile/recovery/errors.spec.ts

```diff
@@ -0,0 +1,114 @@
+import {
+  extractRecoveryCode,
+  gen,
+  MOBILE_URL,
+  website
+} from '../../../../helpers'
+
+context('Mobile Profile', () => {
+  describe('Recovery Flow Errors', () => {
+    before(() => {
+      cy.enableRecovery('code')
+      cy.useConfigProfile('mobile')
+    })
+
+    let email, password
+
+    before(() => {
+      email = gen.email()
+      password = gen.password()
+      cy.registerApi({ email, password, fields: { 'traits.website': website } })
+    })
+
+    describe('code', () => {
+      beforeEach(() => {
+        cy.deleteMail()
+        cy.longCodeLifespan()
+      })
+
+      // Re-enable once mobile supports recovery
+      it.skip('fails with validation errors', () => {
```

Comment:
Please add an issue to track that we need to re-enable these tests!

Reply:
Done https://github.com/ory/kratos/issues/2731